test/test_helper.rb

```diff
@@ -189,9 +189,9 @@ def setup_and_visit_content_item(name, parameter_string = '')
       visit("#{item['base_path']}#{parameter_string}")
     end
   end
-
-  def setup_and_visit_random_content_item(document_type: schema_type)
-    payload = GovukSchemas::RandomExample.for_schema(frontend_schema: schema_type)
+  
+  def setup_and_visit_random_content_item(frontend_schema: schema_type)
+    payload = GovukSchemas::RandomExample.for_schema(frontend_schema: frontend_schema)
```

Comment:
@fofr Following our discussion, I've renamed the document_type param to frontend_schema since that's what it's actually used for. This makes it clearer what we're requesting as a random example.